lib/rules/rule-selector-property-disallowed-list/index.mjs

```diff
@@ -17,24 +19,46 @@ const meta = {
 };
 
 /** @type {import('stylelint').Rule<Record<string, string | RegExp | Array<string | RegExp>>>} */
-const rule = (primary) => {
+const rule = (primary, secondaryOptions) => {
 	return (root, result) => {
-		const validOptions = validateOptions(result, ruleName, {
-			actual: primary,
-			possible: [validateObjectWithArrayProps(isString, isRegExp)],
-		});
+		const validOptions = validateOptions(
+			result,
+			ruleName,
+			{
+				actual: primary,
+				possible: [validateObjectWithArrayProps(isString, isRegExp)],
+			},
+			{
+				actual: secondaryOptions,
+				possible: {
+					ignore: ['keyframe-selectors'],
+					splitList: [isBoolean],
```

Comment:
[suggest] Let's remove `splitList`, which is no added actually.

```suggestion
```

Reply:
Bad copy & paste. Shameful! Heheh.